config/src/main/java/com/alibaba/nacos/config/server/service/repository/RowMapperManager.java

```diff
@@ -47,138 +47,138 @@
  * @author <a href="mailto:liaochuntao@live.com">liaochuntao</a>
  */
 public final class RowMapperManager {
-    
+
```

Comment:
same question as above, do not change indent.

Reply:
ok